md/md.go

```diff
@@ -100,30 +101,138 @@ func (m *MD) renderTable(ctx context.Context, w io.Writer, schema, tableName str
 	canvas.H1(markdown.Code(tableName))
 	canvas.PlainText("")
 	mdTableRows := [][]string{}
+	constraints, err := m.db.ListConstraints(ctx, schema, tableName)
+
+	if err != nil {
+		return fmt.Errorf("fetching constraints for '%s.%s': %w", schema, tableName, err)
+	}
+
 	columns, err := m.db.ListColumns(ctx, schema, tableName)
 	if err != nil {
 		return fmt.Errorf("fetching columns for '%s.%s': %w", schema, tableName, err)
 	}
-	for _, column := range columns {
+	for idx := range columns {
```

Comment:
was it necessary to use `idx` instead of getting the `column` struct directly?

Reply:
The main reason is that I change slice values later in constraints checks. Using column is using a copy of the slice item. Maybe I'm wrong as I'm less acquainted with Golang than with other languages. It looked like it did not work without slice indirection.